### PR TITLE
Fix dependency display in setup wizard: remove tooltip, clickable

### PR DESCRIPTION
**Pull request by kzantow**
Install wizard dependencies were displayed in a tooltip, which was unwieldy in some cases and in general was not a good UX. This PR moves the dependency information to a hidden div until expanded by a clickable link with dependency count visible.

WAS:
image

NOW:
image

### CODE REVIEW

**Comment by kzantow** [comment]
@reviewbybees

**Comment by daniel-beck** [comment]
Supersedes #2009 FYI @tfennelly

**Comment by daniel-beck** [comment]
Does anyone else think it's weird that clicking 'Dependencies' will focus the checkbox for the list item?

**Comment by kzantow** [comment]
@daniel-beck yes, the focus thing is not ideal. It is not entirely trivial to change that, but I will if it's deemed a big enough issue

**Comment by daniel-beck** [comment]
FWIW this makes me concerned that implied dependencies are not considered at all on the setup dialog. If we think they're so important that we don't allow disabling the plugins when an implied dependent plugin is enabled, we should consider them important enough to install them when such a plugin is installed, no? Otherwise this could result in a giant mess of broken plugins.

CC @tfennelly

**Comment by tfennelly** [comment]
:bee I haven't tried it but the screenshots looks great.

Btw ... wouldn't `outline: none;` for `:focus` get rid of the outline?

File: war/src/main/js/templates/pluginSelectionPanel.hbs [hunk 1]
```diff
@@ -40,6 +39,20 @@
...
+							{{../../../translations.installWizard_installIncomplete_dependenciesLabel}}<span class="badge">{{dependencyCount ../plugin.name}}</span>
+						</div>
+						<div class="dep-list">
+							{{#eachDependency ../plugin.name}}
+							<span class="dep badge">
+							{{title}}
+							</span>
+							{{/eachDependency}}
+						</div>
+					</deps>
```
Comment (olivergondza) [inline war/src/main/js/templates/pluginSelectionPanel.hbs]: `deps`?

Comment (kzantow) [inline war/src/main/js/templates/pluginSelectionPanel.hbs]: @olivergondza - short for dependencies? I'll be happy to update, if it's important to you.

Comment (olivergondza) [inline war/src/main/js/templates/pluginSelectionPanel.hbs]: I have no idea how to debug this beast but `</deps>` does not seem to be a valid closing tag. It should be `</span>`.

Comment (kzantow) [inline war/src/main/js/templates/pluginSelectionPanel.hbs]: Oh wow, definitely overlooked that. Too much React on the brain, I suppose. Good catch :)

File: war/src/main/less/pluginSetupWizard.less [hunk 2]
```diff
@@ -342,7 +342,7 @@
 
 		            &:hover{
 		              background: rgba(100, 200, 255, 0.1);
-		              box-shadow:inset 0 200px 200px -200px rgba(100, 200, 255, 0.33),inset 0 0 0 1px rgba(100, 200, 255, 0.33);
+                      box-shadow:inset 0 400px 400px -100px rgba(100, 200, 255, 0.33), inset 0 0 0 1px rgba(100, 200, 255, 0.33);
```
Comment (kzantow) [inline war/src/main/less/pluginSetupWizard.less]: .. there was an actual change here, not just whitespace: made the box shadow overlay expand farther, as the box is larger when it has dependencies expanded

**Comment by daniel-beck** [comment]
This is weird. Not sure I determined it example, but expanding/closing a checked plugin's dependencies list will expand/close all other (checked) plugins' dependencies (possibly only those further down the list).

**Comment by daniel-beck** [comment]
Works like a charm. Thanks @kzantow !

**Pull request merged**